3scale/ansible/playbooks/roles/backend-app/templates/resources.yaml.j2

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
If we're keeping the resource list here we should remove it from the `backend-app` directory, so it's only maintained in one place